src/index.tsx

```diff
@@ -70,15 +70,15 @@ const AppWithProviders = () => {
       <ErrorBoundary>
         <Provider store={store}>
           <QueryClientProvider client={queryClient}>
-            <Boundary>
+            <LoadingBoundary>
```

Comment:
still gotta figure out a better way
boundaries outside of LanguageProvider cant catch errors